src/defu.ts

```diff
@@ -39,9 +40,17 @@ function _defu<T>(
         value,
         object[key],
         (namespace ? `${namespace}.` : "") + key.toString(),
-        merger
+        merger,
+        clone
       );
     } else {
+      if (clone) {
+        // eslint-disable-next-line no-var
+        var structuredClone: (x: any) => any;
+        value = structuredClone
```

Comment:
I think the condition we're looking for it

```suggestion
        value = typeof structuredClone !== undefined
```

still, the comments in https://github.com/unjs/defu/pull/92/files#r1541894626 stand as `structuredClone` wouldn't support all types as well. :(